python/ray/tune/util.py

```diff
@@ -18,6 +20,59 @@
 PINNED_OBJECT_PREFIX = "ray.tune.PinnedObject:"
 
 
+class UtilMonitor(Thread):
+    """Class for system usage utilization monitoring.
+
+    It keeps track of CPU, RAM, GPU, VRAM usage (each gpu separately) by
+    pinging for information every x seconds in a separate thread.
+    """
+    def __init__(self, delay=0.7):
+
+        try:
+            global psutil
+            import psutil
+            global GPUtil
+            import GPUtil
```

Comment:
Could we separate psutil from GPutil? It's much more common for the former to be installed.